bin/helper_upgrade_analyzer

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
yeah, that was ugly, a big hack around docker-compose issue using interpolation in 'command:'.
Mehar made the needs for that file obsolete ( upgrade_analyzer can now use ENV var directly when available )